docs/framework_utils.rst

```diff
@@ -9,6 +9,8 @@ General Utility
 
 .. automodule:: redbot.core.utils
     :members: deduplicate_iterables, bounded_gather, bounded_gather_iter
+.. autoclass:: AsyncIter
+    :members:
```

Comment:
For the methods to show as `async for ... in enumerate(start=)` we need to manually indicate it for these two methods:
![image](https://user-images.githubusercontent.com/6032823/79696560-a17bf600-827d-11ea-8bf3-df493adc13b7.png)

```suggestion

.. autoclass:: AsyncIter
    :members:
    :exclude-members: enumerate, filter

    .. automethod:: enumerate
        :async-for:

    .. automethod:: filter
        :async-for:

```